app/src/main/java/erevacation/com/userlist/ui/homescreen/profile/ProfileFragmentPresenter.kt

```diff
@@ -15,6 +15,7 @@ class ProfileFragmentPresenter @Inject constructor() : ProfileContract.ProfilePr
         this.view = view as ProfileContract.ProfileView
         this.binding = (this.view as?ProfileFragment)?.binding
         buildLayout()
+        publishUserDetails((view as ProfileFragment).user)
```

Comment:
We can add here public method `getUser()` so we can fetch user value. On this way we make user var only readable.

Reply:
Temporary solution, we will fix it when we create class Usecase